### PR TITLE
Fix update sentry

### PR DESCRIPTION
**Pull request by JozoSalt**
After deploy we need to run 
* pip install -r requirements.txt
* sentry --config=config.py upgrade
* restart

### CODE REVIEW

**Comment by jgorset** [comment]
:+1:

File: requirements.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ django-static-compiler==0.3.3
 django-templatetag-sugar==0.1
 django-toolbelt==0.0.1
 email-reply-parser==0.2.0
-gunicorn==0.17.4
+gunicorn==19.2.1
```
Comment (sindrenm) [inline requirements.txt]: Holy balls!

Comment (zenangst) [inline requirements.txt]: Such upgrade

**Comment by mathiasbno** [comment]
:pager: :abcd:

File: requirements.txt (hunk 1 above)
Comment (jgorset) [inline requirements.txt]: Wow.

**Pull request merged**